e2e/features/add-addendum-evidence.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Add addendum evidence
+
+
+  Background:
+    Given I am signed in as a `Legal Rep`
+    And I create a new case
+    And I save my initial appeal
+    And I submit my appeal
+    And I switch to be a `Case Officer`
+    And I request respondent evidence
+    And I upload respondent evidence
+    And I switch to be a `Legal Rep`
+    And I build my case
+    And I submit my case
+    And I switch to be a `Case Officer`
+    And I request respondent review
+    And I add the appeal response
+    And I request hearing requirements
+    And I list the case
+    And I create case summary
+    And I generate the hearing bundle
+    And I start decision and reasons
+
+  @add-addendum-evidence @RIA-1603
+  Scenario: Add addendum evidence
+    When I click the `Documents` tab
+    Then I should see the `Documents` page
+
+    When I click the `Add additional evidence as an addendum` link
+    Then I should see the `Upload additional evidence` page
+    And I should see the text `Add the evidence and the context. This evidence will not be included in the hearing bundle, it will be added as an addendum.`
+    And I should see the text `If you have evidence from both the appellant and the respondent, submit these seperately.`
+    And I should see the text `You’ll need to explain why this evidence is being submitted after the hearing bundle has been produced.`
+
+    When I click the `The appellant` label
```

Comment:
Any chance we can write the DataTable(or) scenario outline to cover _The respondent_ scenario here..

Reply:
yes, I will add "The respondent" scenario

Reply:
👍